src/fetchMOTD.ts

```diff
@@ -43,10 +44,26 @@ export function fetchMOTD(): void {
   } else {
     const origin = utils.addSubdomain(realtimeOrigin, "firebase-public");
     const c = new Client({ urlPrefix: origin, auth: false });
-    c.get("/cli.json").then((res) => {
-      motd = Object.assign({}, res.body);
-      configstore.set("motd", motd);
-      configstore.set("motd.fetched", Date.now());
-    });
+    c.get("/cli.json")
+      .then((res) => {
+        motd = Object.assign({}, res.body);
+        configstore.set("motd", motd);
+        configstore.set("motd.fetched", Date.now());
+      })
+      .catch((err) => {
+        const args = process.argv.slice(2);
+
+        if (args.findIndex((arg) => arg.startsWith("emulators:")) === -1) {
+          throw new FirebaseError(
+            "Unable to fetch the CLI version check configuration. Make sure that you are online and try again.",
+            { original: err }
+          );
+        }
+
+        utils.logLabeledWarning(
+          "emulators",
+          "Unable to fetch the CLI version check configuration, emulator functionality may be incorrect."
+        );
```

Comment:
I think you can trivialize this a lot (I have other context as to why "just ignoring" this error may be desirable):

```suggestion
        utils.logWarning("Unable to fetch the CLI MOTD and remote config.");
        logger.debug(`Failed to fetch MOTD ${err}`);
```

You'll have to import logger for `logger.debug`, but that's easy enough :)